lib/loaders/typescript.js

```diff
@@ -11,6 +11,7 @@
 
 const loaderFeatures = require('../loader-features');
 const babelLoader = require('./babel');
+const forkedTypesPluginUtil = require('../plugins/forked-ts-types');
```

Comment:
That was my first solution. I did not commit it because I was hidding a dependency in the execution path of the library. I guess that since we use it to report back an error it should be fine.

I'll take care of it. thanks!!